mavros_extras/src/plugins/mocap_fake_gps.cpp

```diff
@@ -145,7 +145,7 @@ class MocapFakeGPSPlugin : public plugin::PluginBase
 		pos.eph = 2;
 		pos.epv = 2;
 		pos.fix_type = 3;
-		pos.satellites_visible = 5;
+		pos.satellites_visible = 10;
 		UAS_FCU(m_uas)->send_message_ignore_drop(pos);
 	}
 };
```

Comment:
It is been a while, but I guess there was a minimum number of Sats in px4 firmware to accept GPS data, at least at the time I tested it.